skeleton/template.yaml

```diff
@@ -283,7 +283,7 @@ spec:
           srcRepoURL: https://${{ parameters.githubServer if parameters.hostType === 'GitHub' else parameters.gitlabServer }}/${{ parameters.repoOwner }}/${{ parameters.repoName }}
           argoComponentOverlays: './components/${{ parameters.name }}/overlays'
           owner: ${{ parameters.owner }} 
-          image: quay.io/redhat-appstudio/dance-bootstrap-app:latest # bootstrap app image as placeholder
+          image: quay.io/redhat-ai-dev/ai-template-bootstrap-app:latest # bootstrap app image as placeholder
```

Comment:
is the `values.image` ever been used anywhere?
I searched `values.image` in the repo, seems only the source repo uses that param. the gitops one has been deprecated, and `appContainer` is being used instead?

Reply:
I'm not sure, I was just mirroring the non `redhat-ai-dev` images but if there is unused values it may be worthwhile to do a deep dive on the template as a whole to see if there is unused/redundant pieces that are apart of it.